src/App.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/extensions */
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import './App.css';
+import Navbar from './Component/navibar';
+import Pagecategory from './pages/PageCatego';
+import BooksPage from './pages/PageBook,js';
+import NoMatch from './pages/nomatch,js';
```

Comment:
- [ ] KIndly check your import code you accidentally added a `,` that makes your project not run. Also, ensure all pages are displayed as per the project requirement.

Reply:
Ohh yes. Sorry, thank you for pointing out this mistake. & what does it mean "Also, ensure all pages are displayed as per the project requirement." can you clarify this plz?